docs/src/tutorials/twodimensional_linear_advection.md

```diff
@@ -0,0 +1,176 @@
+# Linear advection equation in two dimensions
+
+In this tutorial, we consider the linear advection equation in two dimensions
+
+```math
+\begin{aligned}
+    \partial_t u(t,x,y) + a_1\partial_x u(t,x,y) + a_2\partial_y u(t,x,y) &= 0, && t \in (0,T), (x, y)\in\Omega \\
+    u(0,x,y) &= u_0(x,y), && (x,y) \in\Omega, \\
+    u(t,x,y) &= g(t,x,y), && t \in (0,T), (x,y) \in\Gamma_-,
+\end{aligned}
+```
+
+Here, ``\Omega`` is a two-dimensional domain, and ``\Gamma_- = \{(x, y)\in\partial\Omega|\boldsymbol{a}\cdot\boldsymbol{n}(\boldsymbol{x}) < 0\}`` is the inflow boundary,
```

Comment:
````suggestion
In this tutorial, we consider the linear advection equation in two dimensions

```math
\begin{aligned}
    \partial_t u(t,x,y) + a_1\partial_x u(t,x,y) + a_2\partial_y u(t,x,y) &= 0, && t \in (0,T), (x, y)\in\Omega, \\
    u(0,x,y) &= u_0(x,y), && (x,y) \in\Omega, \\
    u(t,x,y) &= g(t,x,y), && t \in (0,T), (x,y) \in\Gamma_-.
\end{aligned}
```

Here, ``\Omega`` is a two-dimensional domain, and ``\Gamma_- = \{(x, y)\in\partial\Omega|\boldsymbol{a}\cdot\boldsymbol{n}(\boldsymbol{x}) < 0\}`` is the inflow boundary,
````